### ISSUE TITLE
[Bug] Selected value is not displayed

### ISSUE THREAD

**Issue by nabrezzelt** [Blazored/Typeahead#263]
**Describe the bug**
After setting the Value-Property (click the button) the value is not displayed:

![grafik](https://user-images.githubusercontent.com/21213524/156362933-b5480f1f-1ca4-45b2-8451-f20d7ee3d315.png)

**To Reproduce**
1. Click the Test Button

**Expected behavior**
The Value should be displayed after setting the Value-property manual.

**Screenshots**

**Hosting Model:**
 - Blazor Server
 

**Sample code:**
```razor
<p>@_customer?.FullName</p>
<BlazoredTypeahead TValue="Customer"
                   TItem="Customer"
                   SearchMethod="@SearchCustomer"
                   @bind-Value="_customer"
                   Debounce="500">
    <SelectedTemplate>
        @context.FullName
    </SelectedTemplate>
    <ResultTemplate>
        @context.FullName
    </ResultTemplate>
    <NotFoundTemplate Context="_">
        No customer found.
    </NotFoundTemplate>
</BlazoredTypeahead>

<button @onclick="SetCustomer">test</button>

@code {
    private Customer _customer;

    private void SetCustomer()
    {
        _customer = new Customer();
    }

    private async Task<IEnumerable<Customer>> SearchCustomer(string s)
    {
        return await Task.FromResult(new List<Customer>());
    }

    private class Customer
    {
        public string FullName = "Mayer, Hans";
    }
}
```

**Comment by djukic-marko** [comment]
Hello there :)
I beleve this is a duplicate of this [issue](https://github.com/Blazored/Typeahead/issues/217).
Try downgrading to 4.5.1...

**Comment by benshemmeld** [comment]
@nabrezzelt I have fixed this bug in this PR #278

**Comment by nabrezzelt** [comment]
Hi, thanks for investigating and solving the problem :). I will try it the next days.

And thanks @djukic-marko for the info about the rollback to 4.5.1
